nox/virtualenv.py

```diff
@@ -167,5 +167,8 @@ def create(self):
 
         return True
 
-    def install(self, *args):
+    def install(self, *args, **kwargs):
+        env = kwargs.get('env', None)
+        if env:
+            self.env.update(env)
```

Comment:
My concern here is that whatever env you pass to `install` becomes `sticky` (it's there even in the next call to `install`). Perhaps we should augment `run` to accept an `env` kwarg and pass that all the way down?

Reply:
The modified version of `run` looks like this now:
``` python
def run(self, args, in_venv=True, env=None):
        """Runs a command. By default, the command runs within the
        environment."""
        env_copy = self.env.copy() if in_venv else None
        if env_copy is not None and env is not None:
            env_copy.update(env)
        return Command(
            args=args,
            env=env_copy,
            silent=True,
            path=self.bin if in_venv else None).run()
 ```
But I think I relied on the `sticky`ness of my previous implementation to test it works, how would I go about it now? I don't know much about the way `mock_run.assert_called_with` works

Reply:
Sound good :) I just pushed the changes that I showed here.